src/views/Timesheets/components/SavedShiftsList/SavedShiftsList.module.scss

```
@@ -16,3 +16,26 @@
     }
   }
 }
+_RButton {
```

ditto, and maybe make this a more descriptive name
```suggestion
.RButton {
```